pkg/controller/bootstrap/bootstrap.go

```diff
@@ -183,6 +183,12 @@ func (b *Bootstrap) Run(destDir string) error {
 		configs = append(configs, kconfigs...)
 	}
 
+	if releaseVersion, ok := cconfig.Annotations[ctrlcommon.ReleaseImageVersionAnnotationKey]; ok {
+		if err := ctrlcommon.ProxyValidation(cconfig.Spec.Proxy, releaseVersion, icspRules); err != nil {
```

Comment:
Hmm, just for my own curiosity, this will check via the bootstrap network on the bootstrap node right?

Is there a possibility that the bootstrap network is different? Does it even use the proxy you provide to the cluster?